### PR TITLE
updated samples to use manifest version 1.16

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AppManifest/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
```
Comment (Pawank-MSFT) [inline samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AppManifest/manifest.json:2]: Why we've downgraded, from 1.16 to 1.14

File: samples/tab-app-monetization/nodejs/ClientApp/appManifest/manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.13/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.13",
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
```
Comment (Pawank-MSFT) [inline samples/tab-app-monetization/nodejs/ClientApp/appManifest/manifest.json:2]: Why we have downgraded this, can we use 1.16?

**Review by Pawank-MSFT — COMMENTED** [review]
please review

File: samples/tab-app-monetization/nodejs/ClientApp/appManifest/manifest.json (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/tab-app-monetization/nodejs/ClientApp/appManifest/manifest.json:2]: These sample use rsc permissions, which break in latest manifest version

File: samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AppManifest/manifest.json (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AppManifest/manifest.json:2]: These sample use rsc permissions, which break in latest manifest version

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approvaing!

**Review by Harikrishnan-MSFT — APPROVED** [review]

**Pull request merged**